docs/how-to/how-to-use-virtio-fs-nydus-with-kata.md

```diff
@@ -0,0 +1,17 @@
+# Kata Containers with virtio-fs-nydus
+
+## Introduction
+
+[Research](https://www.usenix.org/conference/fast16/technical-sessions/presentation/harter) shows that time to take for pull operation accounts for 76% of container startup time but only 6.4% of that data is read. so if we can get data on demand (lazyload), it will speed up the container start. [Nydusd](https://github.com/dragonflyoss/image-service) is the project which build image with new format and can get data on demand when container start.
```

Comment:
Duplicated with kata-nydus-design.md, you can create a link to that markdown instead.

Reply:
done